app/charts/combo/combo-line-dual-state.tsx

```diff
@@ -165,7 +165,9 @@ const useComboLineDualState = (
         };
       })
       .filter(truthy);
-    const yAnchor = isMobile ? chartHeight : mean(values.map((d) => d.yPos));
+    const yAnchor = isMobile
+      ? chartHeight + 32
```

Comment:
Why 32? 😄 Maybe this should be `margin.bottom`?

Reply:
ah sorry, still had that there temp.